galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager.ivt/src/main/java/dev/galasa/cicsts/cemt/manager/ivt/CEMTManagerIVT.java

```diff
@@ -50,19 +51,20 @@ public void login() throws Exception {
       
       resourceTerminal.clear();
       resourceTerminal.waitForKeyboard(); 
+      
    }
    
    @Before
    public void installProgramResource() throws CicstsManagerException, TimeoutException, KeyboardLockedException, TerminalInterruptedException, NetworkException, FieldNotFoundException {
```

Comment:
What happens if this program is already installed in the region before this test runs?

Reply:
This should not be the case for just one test case running as we clear up the resources in the After method. If the same test is running in parallel, this could happen, but we could eliminate this by using random strings for our resource names. I will get an update from Mike on where we are with generating random strings.

Reply:
OK cool